_data/members.yml

```diff
@@ -521,6 +521,13 @@
   family: Tsuru
   role: phd
   nationality: jp
+  scholar: "https://scholar.google.com/citations?user=ToS-6x0AAAAJ&hl=en&oi=sra"
+  bio: >
+    received the master's degree in engineering from Osaka University, Japan, in 2021. He started working as Research Assistant at the CNRS-AIST JRL (Joint Robotics Laboratory), IRL, AIST in 2019. His research topic is combining humanoid robots with vision sensing and real-time action planning for automonous humanoid robots.
+  videos:
+    - title: "Online Object Searching by a Humanoid Robot in an Unknown Environment"
+      url: "https://youtu.be/2rr84jnez4k"
+      img: "http://i1.ytimg.com/vi/YGhJDXUlApM/mqdefault.jpg"
```

Comment:
I think that this URL is wrong.
(A thumbnail which shows HRP-4 is shown.)

Reply:
Sorry, I have to upload an image somewhere.
I will ask Harada-Lab to let me use their webserver to upload and save a new thumbnail image.

Reply:
I think the problem is that you put the wrong url for the image. Have a look as how it is down for Stanislas Brossette.
If you need n image hosted, you can put it in `en\assets\pictures\thumbnail`

Reply:
https://i1.ytimg.com/vi/2rr84jnez4k/hqdefault.jpg works

Reply:
Thank you really so much....
I will create a new pull request to replace the URL.
Thank you.